scripts/release.sh

```diff
@@ -391,8 +398,19 @@ main() {
     release_url=$(gh --repo "${gh_repo}" release view "${RELEASE_VERSION}" --json url --jq '.url')
 
     log_warning ""
-    log_warning "WARNING: The GitHub release for ${RELEASE_VERSION} has been created as a draft, please go to ${release_url} and release it."
+    log_warning "WARNING: The GitHub release for ${RELEASE_VERSION} has been created as a draft, please review it at ${release_url}."
     log_warning ""
+
+    # Give a 10 minute timeout to the user to confirm the release.
+    read -p "Release GitHub release for ${RELEASE_VERSION} [y/N]? " -t 600 -r confirm
```

Comment:
I think with a bit more testing / confidence we can simply drop the publishing of the release in draft mode and 10 minute review prompt and simply publish the full release. 

It was a really great milestone to see the script automatically creating the GitHub release for us today so let's just get a bit more confidence in the next round of releases and then revisit it :)

Reply:
I agree. I was hesitant to add this change, but no matter what arguments I'd pass to `gh create release`, the draft release in the edit UI always has the "set as latest release" checkbox ticked. So, the workaround was to publish the release by editing it, while passing all the possible arguments.

Reply:
I'm going to draft this PR then. We can revisit it when we feel confident about automating the release process.